plat/rpi3/aarch64/plat_helpers.S

```diff
@@ -181,3 +182,7 @@ func plat_reset_handler
 	str	w1, [x0, #RPI3_INTC_MBOX_CONTROL_OFFSET]
 	ret
 endfunc plat_reset_handler
+
+func plat_interrupt_type_to_line
```

Comment:
This is not safe in case interrupt is triggered. Why can't any of the standard definitions in plat_gicv2.c be used for rpi3?

Reply:
The rpi3 doesn't have a GIC.

Reply:
Ok, so this function is needed in order to compile. Instead of having this dummy implementation, move it to ``rpi3_common.c`` and call ``panic()``.